js/components/ladderPage/ladder.tsx

```diff
@@ -0,0 +1,5 @@
+import { ReactElement } from "react";
+
+export const Ladder = (): ReactElement => {
```

Comment:
suggestion: could you add barebones testing for this, mostly so we have a place to add more tests when the time comes?